techniques/flash/FLASH4.html

```diff
@@ -11,7 +11,7 @@ <h3>ActionScript 3 combobox with submit button</h3>
          
             <p>This is a basic ActionScript 3 example of a combobox component with  a submit button to redirect the user to a different resource. </p>
          
-         <pre xml:space="preserve">import fl.accessibility.ComboBoxAccImpl;
+         <pre>import fl.accessibility.ComboBoxAccImpl;
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`